docs/operations/comanage-recipes.md

```diff
@@ -0,0 +1,71 @@
+COManage Recipes
+==============================
+
+A collection of step-by-step instructions for common actions for use by administrators of the OSG COManage.
+
+
+Steps for Common COManage Actions
+----
+This section contains some common actions administrators perform on the OSG COManage, and how to do so.
+
+### Provisioning a CO Group in COManage
+In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on Hosts),
```

Comment:
```suggestion

In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on hosts),
```

Adding a linebreak between the heading and content for readability, not sure why hosts was capitalized